### DISCUSSION TITLE
🚀 Feature Request: Add Triggers to Hide Popups

### DISCUSSION THREAD

**Discussion by bombarda-rpd** [elementor/elementor#21550] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

When I uase a popup with an anchor to the top, the popup is always visible and it should hide.

### Describe the solution you'd like

The popup should have more features like "show on load" and "hide on scroll". This is usefull when using a popup with an anchor to the top.

**Comment by nicholaszein** [comment]
Hello @bombarda-rpd,

the ability to **`Trigger`** a Popup when you load the page is already possible. I've edited the title of your request to reflect the other point, when you trigger a **`Hide`** event to hide Popups.

Please refer to the Popup documentation to learn how to trigger a Popup when the page loads:

https://elementor.com/help/what-is-the-difference-between-conditions-triggers-and-advanced-rules/
